truffle/src/com.oracle.truffle.host/src/com/oracle/truffle/host/HostToTypeNode.java

```diff
@@ -140,7 +141,7 @@ protected static Object doGeneric(
         return convertImpl(operand, targetType, genericType, allowsImplementation(context, targetType),
                         isPrimitiveTarget(targetType), context,
                         InteropLibrary.getUncached(operand),
-                        useTargetMapping,
+                        useTargetMapping, HostLibrary.getUncached(),
```

Comment:
Using uncached `HostLibrary.getUncached()` as the method is annotated by `@TruffleBoundary`.